requirements.txt

```diff
@@ -1,5 +1,5 @@
-git+https://github.com/raiden-network/raiden.git@f9bbbc0aeadf2ce1a54f3ed22de948133bc461df
-raiden-contracts==0.33.3
+git+https://github.com/palango/raiden.git@4dca1d2f35aaa94505cf82e3a2b890bf2106b7df
```

Comment:
is the the correct link? It is not raiden-network but palango, now?

Reply:
This was the only way to test this before the PR in raiden is merged, I'll change that back. Just wanted CI to run already.